main.go

```diff
@@ -23,11 +23,18 @@ func scoopHome() (res string) {
 	if value, ok := os.LookupEnv("SCOOP"); ok {
 		res = value
 	} else {
-		var err error
-		res, err = os.UserHomeDir()
+		home, err := os.UserHomeDir()
 		checkWith(err, "Could not determine home dir")
 
-		res += "\\scoop"
+		path := home + "\\.config\\scoop\\config.json"
+		content, err := os.ReadFile(path)
+		if err != nil {
+			res = home + "\\scoop"
+		} else {
+			var parser fastjson.Parser
+			config, _ := parser.ParseBytes(content)
+			res = string(config.GetStringBytes("root_path"))
+		}
```

Comment:
```suggestion
		if content, err := os.ReadFile(path); err == nil {
			var parser fastjson.Parser
			config, _ := parser.ParseBytes(content)
			res = string(config.GetStringBytes("root_path"))
		} else {
			res = home + "\\scoop"
		}
```

I think this inversion of condition will read better